internal/services/applications/parse/redirect_uris.go

```diff
@@ -75,3 +75,17 @@ func (id RedirectUrisId) Segments() []resourceids.Segment {
 func (id RedirectUrisId) String() string {
 	return fmt.Sprintf("Application Redirect URIs (Application ID: %q, URI Type: %q)", id.ApplicationId, id.UriType)
 }
+
+func (id RedirectUrisId) FromParseResult(input resourceids.ParseResult) error {
```

Comment:
This'll need to be a pointer:

```suggestion
func (id *RedirectUrisId) FromParseResult(input resourceids.ParseResult) error {
```